src/options/index.ts

```diff
@@ -19,6 +20,15 @@ export function resolveAttributeSeparatorOption(attributeSeparator: 'always' | '
 	);
 }
 
+export function resolveClosingBracketPositionOption(closingBracketPosition: 'new-line' | 'last-line'): boolean {
+	switch (closingBracketPosition) {
+		case 'new-line':
+			return true;
+		case 'last-line':
+			return false;
+	}
+}
```

Comment:
Since this part of the code is affected by the input from the consumer, we should raise an error (as in line 18).